deploy/requirements.txt

```diff
@@ -1,3 +1,2 @@
 -r ../requirements.txt
-uwsgi
```

Comment:
This needs to be added also to `requirements.in`-file, because `requirements.txt` is derived from that file.

Reply:
This deploy/requirements.txt. It is not handled by pip-tools. Although it should, as whitenoise can change in incompatible ways.
Could we maybe just make whitenoise part the smbackend proper? It should work just fine in local development as well.

Reply:
Ok, let's leave it like this for now.